dali/base/dautils.cpp

```diff
@@ -116,7 +116,7 @@ class CMultiDLFN
             CDfsLogicalFileName lfn;
             lfn.setAllowWild(true);
             lfn.set(s);
-            dlfns.push_back(lfn);
+            dlfns.push_back(std::move(lfn));
```

Comment:
I'm not convinced this is really worth the extra code that needs maintaining.  If this is really a bottleneck, you would be better off implementing a new constructor for   CDfsLogicalFileName and use vector::emplace_back